Source/WebCore/dom/Node.h

```diff
@@ -797,12 +790,14 @@ class Node : public EventTarget, public CanMakeCheckedPtr<Node> {
     mutable uint32_t m_refCountAndParentBit { s_refCountIncrement };
     const uint16_t m_typeBitFields;
     mutable OptionSet<StateFlag> m_stateFlags;
+    mutable OptionSet<ElementStateFlag> m_elementStateFlags;
+    mutable StyleBitfields m_styleBitfields;
 
     CheckedPtr<ContainerNode> m_parentNode;
     TreeScope* m_treeScope { nullptr };
-    CompactPointerTuple<Node*, uint16_t> m_previous;
+    Node* m_previousSibling;
```

Comment:
Shouldn't this be a CheckedPtr?

Reply:
At the very least: `{ nullptr };`? Who is default initializing this otherwise?

Reply:
Definitely these are missing a `{ nullptr }`, good catch - I've added null initialization in the latest push. On changing these to `CheckedPtr`, we discussed this elsewhere and that might be more suitable for a different future patch due to the potential performance impact.